### PR TITLE
Project - Daniel

### PR DESCRIPTION
**Pull request by daniielperez10**

### CODE REVIEW

**Comment by Enchufa2** [comment]
The project is empty! Please commit and push everything under _projects and let me know when this is ready.

**Comment by daniielperez10** [comment]
I think now it should be okay

**Comment by Enchufa2** [comment]
Now the YAML header is not compliant with the instructions in the tutorial, and I only see code and no text. Please see your colleagues' projects. Switching back to draft.

**Comment by daniielperez10** [comment]
ok, I think I was missing toc : true.
sorry for the inconvenience

File: _projects/2023/1004075443/1004075443.Rmd [hunk 1]
```diff
@@ -0,0 +1,401 @@
+---
+title: "The Demography of the World Population from 1950 to 2100"
+description: |
+  A short description of the post.
```
Comment (Enchufa2) [inline _projects/2023/1004075443/1004075443.Rmd]: There is no description, and there is no category.

File: _projects/2023/1004075443/1004075443.Rmd [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+author: Daniel Pérez Gutiérrez
+date: 2023-12-18
```
Comment (Enchufa2) [inline _projects/2023/1004075443/1004075443.Rmd]: This is not the correct format for the date.

File: _projects/2023/1004075443/1004075443.Rmd [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
```
Comment (Enchufa2) [inline _projects/2023/1004075443/1004075443.Rmd:14]: There is no text.

File: _projects/2023/1004075443/1004075443.Rmd [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+#devtools::install_github("hrbrmstr/streamgraph")
+#library(streamgraph)
+library(ggstream)
+library(cowplot)
+library(extrafont)
+#font_import()
+#loadfonts()
+#############################################################################################
+########################################################################################################
+#DATOS PREPARADOS
```
Comment (Enchufa2) [inline _projects/2023/1004075443/1004075443.Rmd]: There are comments in Spanish.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Sorry to be blunt, Daniel, but I insist: there is no text in your post. The projects are expected to be blog posts where you guide the reader through your work. You have a bunch of examples in the webpage.

**Comment by daniielperez10** [comment]
Dear Iñaki, sorry for the confusion, I still don't know if I am uploading things correctly. I hope everything is correct.

Thank you

**Review by Enchufa2 — APPROVED** [review]
Great work, congrats! I added the dataset and clean things up a bit. Merging.

**Comment by Enchufa2** [comment]
And here's your work: https://csslab.uc3m.es/dataviz/projects/2023/100407543/

**Pull request merged**